test/TestServerProjectsLowLevel/dpg-customization/Customization/DPGClient.cs

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Security.Cryptography;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using dpg_customization_LowLevel.Models;
+
+namespace dpg_customization_LowLevel
+{
+    public partial class DPGClient
+    {
+        /// <summary> Get models that you will either return to end users as a raw body, or with a model added during grow up. </summary>
+        /// <param name="mode"> The mode with which you&apos;ll be handling your returned body. &apos;raw&apos; for just dealing with the raw body, and &apos;model&apos; if you are going to convert the raw body to a customized body before returning to users. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="mode"/> is null. </exception>
+        public virtual async Task<Response<Product>> GetModelValueAsync(string mode, CancellationToken cancellationToken = default)
+        {
+            RequestContext requestContext = new RequestContext();
+            requestContext.CancellationToken = cancellationToken;
+
+            Response response = await GetModelAsync(mode, requestContext);
+            return Response.FromValue((Product)response, response);
+        }
+
+        /// <summary> Get models that you will either return to end users as a raw body, or with a model added during grow up. </summary>
+        /// <param name="mode"> The mode with which you&apos;ll be handling your returned body. &apos;raw&apos; for just dealing with the raw body, and &apos;model&apos; if you are going to convert the raw body to a customized body before returning to users. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="mode"/> is null. </exception>
+        public virtual Response<Product> GetModelValue(string mode, CancellationToken cancellationToken = default)
```

Comment:
With the updated guidance from https://github.com/Azure/autorest.csharp/issues/2014, we will not use Value in the name.

Reply:
After discussion, we will still use the Value suffix as a suggestion for grow-up scenarios.